tests/libnode/node.go

```diff
@@ -333,6 +333,16 @@ func GetNodeWithHugepages(virtClient kubecli.KubevirtClient, hugepages k8sv1.Res
 	return nil
 }
 
+func NodeNameWithHandler() string {
+	listOptions := k8smetav1.ListOptions{LabelSelector: v1.AppLabel + "=virt-handler"}
```

Comment:
Just my personal pref but I would inline this.

Reply:
There are plenty of bad things in this function: it returns a random node, making an uninformed scheduling decision for the rest of the test using it. It uses `Except` instead of returning an `error`. It's name is awkward. Of all things, I don't mind the intermediatery variable.

I prefer to consider these in a follow-up PR, but reviewer is king.

Reply:
Right, separation and moving these funcs closer to where they are used is a good first step. Are you going to create an issue for the follow ups?

Reply:
I've modified the name and inlined the variable per your request. The usage of `Expect` in libraries is too ubiquitous to handle right now.